### PR TITLE
Implement admin view product

### PR DESCRIPTION
**Pull request by nhuthh-2546**
image

### CODE REVIEW

**Comment by nhuthh-2546** [comment]
https://edu-redmine.sun-asterisk.vn/issues/37855

**Comment by nhuthh-2546** [comment]
image

**Comment by nhuthh-2546** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
   
   namespace :admin do
     resources :orders
+    resources :products
```
Comment (vanvtt-0952) [inline config/routes.rb]: thêm `only` vào cụ

**Comment by nhuthh-2546** [comment]
ready

**Pull request merged**